### ISSUE TITLE
Allow users to pass hyperparameters to Legion training API

### ISSUE THREAD

**Issue by kirillmakhonin** [legion-platform/legion#480]
This is subtask of #930

Legion toolchain should provide ability to:

- Declare training hyperparameters with default values (e.g. count of layers in neural network)
- Use values from ModelTraining CR
- Provide values for training in local mode

This hyperparameters with values should be sent with train metrics as labels.

**Comment by mcdoker18** [comment]
Implemented #988
